src/pages/components/loading/usage.mdx

```diff
@@ -41,23 +44,256 @@ and help to notify users that loading is underway.
+  overwhelm users.
+
 ## Formatting
 
-Loading spinners may be scaled down if the loading experience is contextual to a
-certain item on the page.
+### Anatomy
+
+<Row>
+<Column colLg={8}>
+
+<Tabs>
+
+<Tab label="Large">
+
+![Large loading component anatomy](images/loading-usage-anatomy-large.png)
+
+</Tab>
+
+<Tab label="Small">
+
+![Small loading component anatomy](images/loading-usage-anatomy-small.png)
+
+</Tab>
+
+</Tabs>
+
+</Column>
+</Row>
+
+**1. Loading indicator:** A circular animated icon indicating that a process is
+underway. It serves as the primary visual cue for the loading state.
+
+### Sizing
+
+The loading component is available in two sizes: **large** and **small**. The
+large loading indicator size is the default size.
+
+<Row>
+<Column colLg={8}>
+
+![Large and small sizes of the loading component](images/loading-usage-sizing.png)
+
+</Column>
+</Row>
+
+<Caption>Large and small sizes of loading component</Caption>
+
+#### Large size
+
+The large loading indicator is most commonly used to display loading in
+full-screen takeover situations. It can also appear on sections within a page,
+in [modals](/components/modal/usage/#loading), and on individual
+[tiles](/components/tile/usage). A background overlay is typically placed behind
+the large loading indicator so the user cannot interact with the elements
+currently in the loading state.
+
+<Row>
+
+<Column colLg={8}>
+<GifPlayer color='light'>
+
+![Example of the large loading component in a UI](/images/loading-usage-content-large.gif)
+
+![Example of the large loading component in a UI](/images/loading-usage-content-large.png)
+
+</GifPlayer>
+</Column>
+</Row>
+
+<Caption>Example of the large loading component in a UI</Caption>
+
+#### Small size
+
+The small loading indicator is best suited for contextual or localized loading
+experiences within a specific UI element, particularly in inline loading
+scenarios. It works well for quick-loading tasks and can be placed within or
+near the triggering element—such as a
+[button](/components/button/usage/#loading)—to indicate that a process is
+underway. Common examples include saving after clicking on a button, uploading a
+file, or placing the indicator beside a label text to communicate the loading
+status. For detailed guidance, refer to the inline loading component
+[Usage](/components/inline-loading/usage/) tab.
+
+<Row>
+
+<Column colLg={8}>
+<GifPlayer color='light'>
+
+![Example of the small loading component in a UI](/images/modal-error.gif)
+
+![Example of the small loading component in a UI](/images/modal-error.png)
+
+</GifPlayer>
+</Column>
+</Row>
+
+<Caption>Example of the small loading component in a UI</Caption>
+
+### Placement
+
+#### Large loading placement
+
+The large loading indicator is commonly used for full-screen and component-level
+loading scenarios.
+
+For full-screen loading, place the indicator at the center of the viewport and
```

Comment:
I feel like this sentence could be moved into the paragraph above to make this section less choppy.